src/Foundation/NSValue.cs

```diff
@@ -64,6 +68,31 @@ public static NSValue FromPointF (PointF point)
 			get { return (PointF)CGPointValue; }
 		}
 #endif
+
+#if MONOMAC
+		// @encode(CGAffineTransform) -> "{CGAffineTransform=dddddd}" but...
+		// using a C string crash on macOS (while it works fine on iOS)
+		[DllImport ("__Internal")]
+		extern static IntPtr xamarin_encode_CGAffineTransform ();
+
+		// The `+valueWithCGAffineTransform:` selector comes from UIKit and is not available on macOS
+		public unsafe static NSValue FromCGAffineTransform (CGAffineTransform tran)
```

Comment:
Style: complete word as parameter name - `transform`

Reply:
yeah, I thought about it - but it's the current name for iOS/tvOS/watchOS :(
so I decided to be consistent